doc/API.md

```diff
@@ -73,6 +76,98 @@ struct {
 ```
 'block' is a block. The struct is defined in types/block.go.
 
+Explorer
+--------
+
+Queries:
+
+* /explorer
+* /explorer/hash
+
+### /explorer
+
+Function: Returns the status of the blockchain and some
+statistics. All Siacoin amounts are given in Hastings
+
+Parameters: None
+
+Response:
+```
+struct {
+	height            types.BlockHeight (uint64)
+	block             types.Block
+	target            types.Target    (byte array)
+	difficulty        types.Currency  (string)
+	maturitytimestamp types.Timestamp (uint64)
+	circulation       types.Currency  (string)
+
+	transactioncount          uint64
+	siacoininputcount         uint64
+	siacoinoutputcount        uint64
+	filecontractcount         uint64
+	filecontractrevisioncount uint64
+	storageproofcount         uint64
+	siafundinputcount         uint64
+	siafundoutputcount        uint64
+	minerfeecount             uint64
+	arbitrarydatacount        uint64
+	transactionsignaturecount uint64
+
+	activecontractcount uint64
+	activecontractcost  types.Currency (string)
+	activecontractsize  types.Currency (string)
+	totalcontractcost   types.Currency (string)
+	totalcontractsize   types.Currency (string)
+}
+```
+
+### /explorer/block
+
+Function: Return a block at a given height.
+
+Parameters:
+```
+height types.BlockHeight (uint64)
+```
+
+Response:
+```
+struct {
+	block api.ExplorerBlock
+}
+```
+
+### /explorer/hash
+
+Function: Return information about an unknown hash.
+
+Parameters:
+```
+hash string
+```
+`hash` is a hash that is relevant to the Sia network. It can be a block id,
+transacton id, address, unlock hash, siacoin output id, file contract id, or
+siafund output id. The daemon will determine which type of hash was presented
+(if the hash is known at all).
+
+Response:
+```
+struct {
+	 hashtype     string
+	 block        api.ExplorerBlock
+	 blocks       []api.ExplorerBlock
+	 transaction  api.ExplorerTransaction
+	 transactions []api.ExplorerTransaction
```

Comment:
they are not, but they point you to the source file where they get defined. I don't think you need to be more concrete than that. Similarly, we don't define types.Block in API.md.